packages/block-library/src/classic/editor.scss

```diff
@@ -116,6 +116,65 @@
 		cursor: default;
 		border: 2px dashed rgb(186, 186, 186);
 	}
+
+	/**
+	* The following gallery styles were replicated
+	* from the styles applied in the tinymce skin,
+	* /wp-includes/js/tinymce/skins/wordpress/wp-content.css.
```

Comment:
Should we load this file instead? Are these the only styles in this file. I wonder if there are other hidden issues due to the fact that we're not loading this file.

Reply:
This file contains very generic selectors e.g: body etc, loading o the classic editor is fine because it is applied to the content inside an iframe, loading it in Gutenberg may break the things. Loading it would be possible if we could scope all the contents inside the .wp-block-freeform.block-library-rich-text__tinymce class I'm not sure if we have a mechanism available to do that.